generator/src/config.js

```diff
@@ -1,8 +1,10 @@
 import * as path from "path";
+import { pathToFileURL } from "url";
 
 export async function resolveConfig() {
+  const configPath = path.join(process.cwd(), "elm-pages.config.mjs");
   const initialConfig = await await import(
```

Comment:
~~Just me being creepy, dad 😅~~
Saw this on the GH homepage, took a look becuase I was curious.

```suggestion
  const initialConfig = await import(
```